src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -356,6 +356,9 @@
   - name: --enable-managed-identity
     type: bool
     short-summary: Using a system assigned managed identity to manage cluster resource group.
+  - name: --node-osdisk-diskencryptionset-id
+    type: string
+    short-summary: ResourceId of the disk encryption set to use for enabling encryption at rest.
```

Comment:
According to your short-summary, I don't think `node-osdisk` is mandatory. Maybe we should refine this short-summary as well.

Reply:
it should be on `node osdisk` in summary, already fixed